docs/wiki/dev-guides/Release-Process.md

```diff
@@ -289,12 +148,12 @@ page](https://github.com/Point72/csp/releases).
 #### A developer's first release
 
 If this is your first release, you will need an account on pypi.org and
-your account will need to be added as a maintainer to the `csp` project
-on pypi. You will also need to have two factor authentication enabled on
+your account will need to be added as a maintainer to the CSP project
```

Comment:
can we comment on diffs if you are at point72 vs not?
we wont do this from a fork, so I think it needs to be
git pull origin main --tags

also needed git push origin main --tags instead of git push upstream main --follow-tags

Reply:
done